### PR TITLE
Fix Sapper on Windows

### PR DESCRIPTION
**Pull request by benmccann**
Fixes https://github.com/sveltejs/sapper/issues/1333
Fixes https://github.com/sveltejs/sapper/issues/1256

It might only affect some versions of Windows since the tests were passing on GitHub Actions without this change

Thanks to @shackless for remote debugging this with me and testing!

### CODE REVIEW

**Comment by benmccann** [comment]
Added a test

File: test/unit/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as assert from 'assert';
+import { normalize_path } from '../../src/utils';
+
+describe('normalize_path', () => {
+	it('lowercases the first letter', () => {
```
Comment (antony) [inline test/unit/utils.test.ts:5]: Awesome!

I think the `assert` signature is `assert.equal(actual, expected)` though, this is backwards.

Comment (benmccann) [reply test/unit/utils.test.ts:5]: Ah, thanks. That's my Java/JUnit muscle memory. Updated!

Comment (Conduitry) [reply test/unit/utils.test.ts:5]: The way it is now looks like the proper actual, expected ordering to me. However, the strings themselves in the tests need to have the backslashes escaped.

Comment (benmccann) [reply test/unit/utils.test.ts:5]: good catch! updated

**Pull request merged**